torch/csrc/jit/ir/ir.cpp

```diff
@@ -1595,6 +1595,21 @@ Node* Graph::createTupleSlice(Value* tup, int64_t beg, int64_t end) {
   return n;
 }
 
+Node* Graph::createEnumName(Value* e) {
+  assert(e->type()->cast<EnumType>());
```

Comment:
nit: remove assert and use `-type()->expect`

Reply:
Ah, thanks. Didn't know that. Done.